language/move-model/src/lib.rs

```diff
@@ -394,9 +395,9 @@ fn script_into_module(compiled_script: CompiledScript) -> CompiledModule {
 
         struct_defs: vec![],
         function_defs: vec![main_def],
-    }
-    .freeze()
-    .unwrap()
+    };
+    BoundsChecker::verify_module(&module).expect("invalid bounds in module");
```

Comment:
I don't think this is needed. But could defer to @wrwg. Wolfgang, do want these modules verified?